pkg/transform/spanner_test.go

```diff
@@ -125,3 +125,52 @@ func TestSpanNesting(t *testing.T) {
 	b = makeSpanWithTimings(10000, 30000, 30000)
 	assert.False(t, (&a).Inside(&b))
 }
+
+func makeHTTPInfo(method, path, peer, host, comm string, peerPort, hostPort uint32, status uint16, durationMs uint64) httpfltr.HTTPInfo {
+	var i httpfltr.HTTPInfo
+	i.Type = 1
+	i.Method = method
+	i.Peer = peer
+	i.URL = path
+	i.Host = host
+	i.ConnInfo.D_port = uint16(hostPort)
+	i.ConnInfo.S_port = uint16(peerPort)
+	i.Status = status
+	i.StartMonotimeNs = durationMs * 1000000
+	i.EndMonotimeNs = durationMs * 2 * 1000000
+	i.Comm = comm
+
+	return i
```

Comment:
To be more idiomatic, I'd replace this by:
```go
return httpfltr.HTTPInfo {
	Type: 1,
	Method: method,
	Peer: peer,
	URL: path,
	Host: host,
	ConnInfo: bpfConnectionInfoT {
		D_port: uint16(hostPort),
		S_port: uint16(peerPort),
	},
	Status: status,
	StartMonotimeNs: durationMs * 1000000,
	EndMonotimeNs: durationMs * 2 * 1000000,
	Comm: comm,
}
```

Reply:
I can improve the code, but it's a bit tricky since those structs are private.